packages/backend/services/ticket/user.ts

```diff
@@ -158,7 +183,9 @@ const userServiceTicket = new UserService(
                 const tenantId = connection.t_id;
 
                 if (
-                    ((thirdPartyId !== TP_ID.jira && !fields) || (thirdPartyId !== TP_ID.bitbucket && !fields)) &&
+                    ((thirdPartyId !== TP_ID.jira && !fields) ||
+                        (thirdPartyId !== TP_ID.bitbucket && !fields) ||
+                        (thirdPartyId !== TP_ID.github && !fields)) &&
                     fields &&
                     !fields.listId
                 ) {
```

Comment:
@manish-singh-bisht can you elaborate on what you are trying to achieve here ?

Reply:
if listid is not provided for integrations other than these the error will be thrown.